components/Header.tsx

```
@@ -0,0 +1,40 @@
+"use client";
+import React from "react";
+import { usePathname, useRouter } from "next/navigation";
+
+import { Button } from "@/components/ui/button";
+import { ArrowLeft } from "lucide-react";
+import { type Header } from "@/types";
+import PurpleIcon from "@/components/icons/PurpleIcon";
+import LighteningIcon from "@/components/icons/LighteningIcon";
+
+const Header = ({ user }: Header) => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Header'. Consider to delete it or rename it.

'Header' is defined here:


(lint/suspicious/noRedeclare)